video-library/src/App.js

```diff
@@ -0,0 +1,40 @@
+import "./App.css";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Main from "./components/Main/Main";
+import Header from "./components/Header/Header";
+import Trending from "./components/Trending/Trending";
+import Watch from "./components/Watch/WatchVideo";
+
+import LandingPage from "./components/LandingPage/LandingPage";
+import Liked from "./components/Liked/Liked";
+import WatchLater from "./components/WatchLater/WatchLater";
+import HistoryComp from "./components/History/HistoryComp";
+import PlaylistContainer from "./components/Playlist/PlaylistContainer";
+function App() {
```

Comment:
I have a suggestion here, to make import statements cleaner, you can add index.js in the components folder